docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Automating Your Data Pipelines | Dagster
+description: This guide illustrates how to automate your data pipelines
+---
+
+# Automating your data pipelines
+
+In this guide, we’ll walk you through how to run your data pipelines without manual intervention, i.e. automate, and identify the Dagster tools to make that happen.
+
+This guide assumes you have some familiarity with several Dagster concepts, including [software-defined assets](/concepts/assets/software-defined-assets) and [jobs](/concepts/ops-jobs-graphs/jobs).
+
+Dagster offers a few different ways of automating data pipelines, and choosing the right Dagster tool depends on your specific needs. In this guide we'll walk through a few different cases, and which Dagster tool you should use in each case.
+
+---
+
+## My data should be updated every morning
+
+You want some data to be updated every day by a certain time to refresh a dashboard, have updated data go to to a system or you want your database to have the most recent information for the team to work with. This is one of the most traditional cases when building a pipeline. Lets go into how to do this with Dagster!
```

Comment:
```suggestion
You want some data to be updated every day by a certain time to refresh a dashboard, have updated data go to to a system or you want your database to have the most recent information for the team to work with. This is one of the most traditional cases when building a pipeline. Let's go into how to do this with Dagster!
```

🔧 Sorry for not catching this in Notion, but this should be the contraction form and not possessive